### PR TITLE
Branding rollout: Change WordPress.org assets for WCPay 

### LINKED ISSUE #6669: Branding rollout: Change WordPress.org logo for WCPay
### Description
As part of the WooPayments branding rollout, we need to change the following assets of the plugin:
- icons
- banners
- images
- readme.txt (text in the plugin page https://wordpress.org/plugins/woocommerce-payments/)
- adjust logo padding
- change Woo Payments to WooPayments as described in https://github.com/Automattic/woocommerce-payments/issues/6339

### Designs
- paJDYF-7Sw-p2#comment-19439
- 6VfBF8B9vw15Y7897Y6V0b-fi-28_30

### Dev notes
They all are located at https://github.com/Automattic/woocommerce-payments/tree/develop/wordpress-org-assets.
How to release - PaJDYF-933-p2

### Additional context
paJDYF-7Sw-p2#comment-19082

### PR DESCRIPTION
**Pull request by oaratovskyi**
Fixes #6669 #6339

#### Changes proposed in this Pull Request

This PR includes the following changes to be shown on the plugin page  https://wordpress.org/plugins/woocommerce-payments/:
- icons, banners, screenshots
- readme.txt

Also, it includes the following changes:
- Change Woo Payments occurrences to WooPayments as described in #6339 
- Small logo fix

#### Testing instructions

* Check out `update/6669-change-wordpress-assets-to-match-new-branding`
* Onboard new PO account. You'll have to remove the existing one and clear the cache if you have one.
* Check the PO eligibility modal. The text should be WooPayments instead of Woo Payments.
   image
* Check the WooPayments logo on the settings page. It should have equal padding from the top, left, and bottom.
   image
* New images in `wordpress-org-assets` should match the design
* Changes in readme.txt and other text occurrences should make sense and contain new brand name (WooPayments)

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _'QA Testing Not Applicable'_
- [x] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [x] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6766` or branch name `update/6669-change-wordpress-assets-to-match-new-branding` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=update/6669-change-wordpress-assets-to-match-new-branding) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: da8f78444370e5659a110af90b669c3379929721
- Build time: 2023-07-14 11:42:55 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: client/connect-account-page/strings.tsx [hunk 1]
```diff
@@ -111,19 +116,27 @@ export default {
...
+				__(
+					"If you're setting up %s for someone else, it's best to have that person complete the account creation process. As you can see above, we ask for very specific information about the business owner - and you might not have all the details at hand. It's not always possible to change account information once it's been saved, especially if the site accepts live transactions before the correct account information is entered.",
+					'woocommerce-payments'
+				),
+				'WooPayments'
 			),
 		},
 		isMyDataSafe: {
 			heading: __(
 				'Is my data safe with WooCommerce?',
```
Comment (vladolaru) [inline client/connect-account-page/strings.tsx]: I think this string is wrong. It probably should be "Is my data safe with WooPayments?"

**Review by vladolaru — APPROVED** [review]
Tests check out, and the code changes make sense. I left one comment about a string that is probably wrong, but I am approving it. Great work, Oleks!

File: client/connect-account-page/strings.tsx (hunk 1 above)
Comment (oaratovskyi) [reply client/connect-account-page/strings.tsx]: Good point! Fixed in https://github.com/Automattic/woocommerce-payments/pull/6766/commits/da8f78444370e5659a110af90b669c3379929721

**Comment by oaratovskyi** [comment]
Assets deployed.

**Pull request merged**